src/Middlewares/PermissionMiddleware.php

```diff
@@ -17,10 +17,12 @@ public function handle($request, Closure $next, $permission)
             ? $permission
             : explode('|', $permission);
 
-        if (! Auth::user()->hasAnyPermission(...$permission)) {
-            abort(403);
+        foreach ($permission as $p) {
```

Comment:
Let's rename `$permission` to `$permissions (this should be done on line 16 as well).

Generally speaking I don't like abbreviations. Could you rename `$p` to `$permission`?

Reply:
Agreed.

Reply:
Actually `can` can accept array so no need to loop permissions. We can just replace `hasAnyPermission` to `can`. Like so, `if (! Auth::user()->can(...$permission)) {`

Reply:
True. Thanks.